### PR TITLE
Plink2

### PR DESCRIPTION
**Pull request by jmannau**
I have updated the plink2 formula to build the latest stable version (1.90 b3.46) as described by this link https://www.cog-genomics.org/plink2
Note: there isn't a publish tar.gz archive for the 3.46 beta source, so I have linked to the appropriate commit source archive https://github.com/chrchang/plink-ng/commit/96f3637b62f4fddcf1e5a16929b77c637867d8cd
This does result in a audit error, but as there is not an official tarsal archive of this version available.

### Have you:

- [X] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [X] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [X] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [X] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [X] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [X] Removed the `revision` line, if any, when bumping the version number?
- [X] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [X] Not altered the `bottle` section?
- [X Checked that the tests still pass?

### CODE REVIEW

**Comment by iMichka** [comment]
Hello. Thanks for you contribution. Please use tarballs instead of zip files.
See the bot:
> * Stable: Use GitHub tarballs rather than zipballs

File: plink2.rb [hunk 1]
```diff
@@ -18,9 +15,13 @@ class Plink2 < Formula
     sha256 "912423fe36c51e093ee43ae8d35382a1ebdff619ce197a88ff0193498f27e85b" => :mountain_lion
   end
 
+  depends_on :fortran
+  depends_on "openblas" => :optional
+  depends_on "homebrew/dupes/zlib"
```
Comment (jonchang) [inline plink2.rb]: macOS already provides `zlib`, are you certain that linking to e.g., `/usr/include/zlib.h` doesn't work?

File: plink2.rb [hunk 2]
```diff
@@ -1,14 +1,11 @@
 class Plink2 < Formula
-  url "https://github.com/chrchang/plink-ng/archive/v1.90b3.tar.gz"
-  version "1.90b3"
+  desc "PLINK is a free, open-source whole genome association analysis toolset."
+  homepage "https://www.cog-genomics.org/plink2"
+  url "https://github.com/chrchang/plink-ng/archive/96f3637b62f4fddcf1e5a16929b77c637867d8cd.zip"
```
Comment (jonchang) [inline plink2.rb]: Can you please file an issue in the chrchang/pling-ng repository and ask them to tag a release? We'd rather not make up versions like this if possible.

File: plink2.rb [hunk 3]
```diff
@@ -1,14 +1,11 @@
 class Plink2 < Formula
-  url "https://github.com/chrchang/plink-ng/archive/v1.90b3.tar.gz"
-  version "1.90b3"
+  desc "PLINK is a free, open-source whole genome association analysis toolset."
```
Comment (jonchang) [inline plink2.rb]: No need to include the formula name in the description as `brew desc` already does that for you.

File: plink2.rb (hunk 1 above)
Comment (jmannau) [inline plink2.rb]: Plink requires lib v 1.2.11  (https://github.com/chrchang/plink-ng/blob/master/plink_first_compile) hence the dependancy on zlib

File: plink2.rb (hunk 2 above)
Comment (jmannau) [inline plink2.rb]: I have created an issue https://github.com/chrchang/plink-ng/issues/40 to see if we can build from tagged releases.

Comment (ilovezfs) [inline plink2.rb]: A tag was created!

**Comment by jonchang** [comment]
@jmannau upstream has tagged a new commit, please fix the merge conflict, rebase onto master, and use the new tagged version. Once you do that and the tests pass it should be ready to merge. Feel free to ping me if you need help with any of these steps.

**Comment by jmannau** [comment]
@jonchang I have updated with the new tagged release. Sorry for the slow response. Fingers crossed it is good to go now.  🤞

**Pull request closed without merging**